src/index.html

```diff
@@ -1,17 +1,107 @@
 <!doctype html>
-<html lang="en">
+  <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+    onsubmit="onSubmit()">
+      <fieldset id="Personal-info" class="field-container">
+        <legend>Personal information</legend>
+        <label for=""
+               class="form-field">
+          Surname:
+          <input type="text">
+        </label>
+        <label for="" class="form-field">
+          Name:
+          <input type="text">
+        </label>
+        <label for="" class="form-field">
+          How old are You?
+          <input type="number" value="12"
+          min="12" max="100">
```

Comment:
```suggestion
          <input
            type="number"
            value="12"
            min="12"
            max="100"
          >
```
fix attribute indentation in all cases